sdk/core/Azure.Core/tests/public/Samples/SerializationSamples.cs

```diff
@@ -0,0 +1,178 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
It would be good to add tests and samples covering the interop with DynamicData.  These are sketched out at a high level here: https://gist.github.com/annelo-msft/6b079c712bbfed195dd7f726ad53e749

Reply:
Will be addressed in: https://github.com/Azure/azure-sdk-for-net/issues/37709